ui/apps/platform/src/Containers/Dashboard/PatternFly/DashboardPage.tsx

```diff
@@ -40,7 +41,10 @@ function DashboardPage() {
             </PageSection>
             <Divider component="div" />
             <PageSection>
-                <Grid hasGutter>
+                <Grid hasGutter style={{ gridAutoRows: 'max-content' }}>
```

Comment:
This will autosize the grid rows to be the height of the largest item in that row, getting rid of the pesky few pixel difference that is bound to pop up as more widgets are added.